lib/jsdom/living/helpers/style-rules.js

```diff
@@ -92,6 +92,12 @@ exports.propertiesWithResolvedValueImplemented = {
     inherited: false,
     initial: "invert",
     computedValue: "computed-color"
+  },
+  // https://drafts.csswg.org/css-fonts-4/#font-family-prop
+  "font-family": {
+    inherited: true,
+    initial: "",
```

Comment:
It looks like this needs to be set to something like `"serif"`. Can you do that, and then update the test? https://jsbin.com/tulemoreka/edit?html,output